src/Core/src/Platform/Android/MauiApplication.cs

```diff
@@ -40,4 +36,18 @@ void ConfigureNativeServices(HostBuilderContext ctx, IServiceCollection services
 		{
 		}
 	}
+
+	public abstract class MauiApplication : Android.App.Application
+	{
+		protected MauiApplication(IntPtr handle, JniHandleOwnership ownership) : base(handle, ownership)
+		{
+			Current = this;
+		}
+
+		public static MauiApplication Current { get; private set; } = null!;
```

Comment:
Black magic with `null!` where we swear that `null` is not `null` XD. But, in the real world this will always have a value since the app only can ever access this property once it has been set.